### PR TITLE
onboarding tasks

### PR DESCRIPTION
**Pull request by riddhimanpaul0**
# Strategy Onboarding

@probro27

@rodrigotiscareno

### CODE REVIEW

**Comment by riddhimanpaul0** [comment]
@probro27

@rodrigotiscareno

**Comment by ryan-lam** [comment]
@riddhimanpaul0 Test failures: https://github.com/uw-midsun/strategy-onboarding/actions/runs/5010271804/jobs/8979992958?pr=40

File: task2_coding_exercises/q1_arrays.py [hunk 1]
```diff
@@ -49,4 +63,20 @@ def most_freq_element(arr: list[int]) -> int:
...
+    count = {}
+
+    #store how many of each number appears in a dictionary
+    for i in range(len(arr)):
+        if arr[i] in count:
+            count[arr[i]] += 1
+        else:
+            count[arr[i]] = 1
+        
+        max = 0
```
Comment (ryan-lam) [inline task2_coding_exercises/q1_arrays.py:75]: wrong indentation?

Comment (riddhimanpaul0) [reply task2_coding_exercises/q1_arrays.py:75]: Sorry I meant to write the function like 


```
def most_freq_element(arr: list[int]) -> int:

    # insert code here
    count = {}
    #store how many of each number appears in a dictionary
    for i in range(len(arr)):
        if arr[i] in count:
            count[arr[i]] += 1
        else:
            count[arr[i]] = 1
            
    max = 0
    maxList = []
    for key in count:
        if count[key] > max:
            max = count[key]
            maxList.append(key)

    return maxList[0]
```

**Comment by ryan-lam** [comment]
Onboarding lgtm; closing pr

**Pull request closed without merging**